README.md

```diff
@@ -12,6 +12,7 @@
 [![DepShield Badge](https://depshield.sonatype.org/badges/vdaas/vald/depshield.svg?style=flat-square)](https://depshield.github.io)
 [![FOSSA Status](https://app.fossa.com/api/projects/custom%2B21465%2Fvald.svg?type=small)](https://app.fossa.com/projects/custom%2B21465%2Fvald?ref=badge_small)
 [![DeepSource](https://static.deepsource.io/deepsource-badge-light-mini.svg)](https://deepsource.io/gh/vdaas/vald/?ref=repository-badge)
```

Comment:
```suggestion
```
Should we remove this line?

Reply:
I think we should not remove this line because we have already updated deepsource settings, and we can access deepsource UI 👍 

ref: https://deepsource.io/gh/vdaas/vald/?ref=repository-badge

Reply:
https://github.com/vdaas/vald/pull/1881/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R15
:sweat_smile:

Reply:
📝 we are using two deepsource batches. so we will keep it.